packages/app/src/app/components/SubTitle/index.ts

```diff
@@ -1,15 +1,20 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import delayEffect from '@codesandbox/common/lib/utils/animation/delay-effect';
 
-export default styled.h2`
-  ${props => props.delay != null && delayEffect(props.delay || 0)};
-  text-align: center;
-  width: 100%;
-  font-size: 1.75rem;
-  color: ${props =>
-    props.theme.light ? 'rgba(0, 0, 0, 0.7)' : 'rgba(255, 255, 255, 0.7)'};
-  font-weight: 300;
-  margin-top: 0;
-  margin-bottom: 1.5rem;
-  line-height: 1.4;
+const SubTitle = styled.h2`
+  ${({ delay = 0, theme }: { delay: number | null; theme: any }) => css`
```

Comment:
@Saeris I think it's better to extract the `SubTitleProps` here, just to be consistent